src/components/Home/Home.jsx

```diff
@@ -29,10 +31,11 @@ const Home = () => {
           className="u-flex u-flex-justify-center u-mt-2 u-h-5"
         />
       ) : data && data.length === 0 ? (
-        <CompositeHeader
+        <Empty
```

Comment:
Does it have something related to the Stepper? 🤔  This commit should be split. Also, I though that joel told us that it was not an `Empty` component?

Reply:
- En effet c'est un oubli de ma part
- Après les derniers échanges en fin de semaines dernières avec Joel, il m'a confirmé que finalement c'était bien un composant Empty ici.